include/data_fifo.h

```diff
@@ -150,6 +150,18 @@ int data_fifo_num_used_get(struct data_fifo *data_fifo, uint32_t *alloced_num,
  */
 int data_fifo_empty(struct data_fifo *data_fifo);
 
+/**
+ * @brief Uninitialise the data_fifo.
```

Comment:
```suggestion
 * @brief Deinitialize data_fifo.
```
We follow American English (`z`).